diff --git a/jhbuildrc-gtk-osx b/jhbuildrc-gtk-osx
index bcc897d..d2e3b8b 100644
--- a/jhbuildrc-gtk-osx
+++ b/jhbuildrc-gtk-osx
@@ -382,8 +382,10 @@ autogenargs=''
 # Extra cmake args.
 # CMAKE_IGNORE_PREFIX_PATH prevents cmake finding libraries from popular
 # package-management system like Homebrew and MacPorts by default.
+# CMAKE_FIND_FRAMEWORK=LAST makes cmake find frameworks after standard
+# libraries or header in find_* commands
 #
-cmakeargs = '-DCMAKE_IGNORE_PREFIX_PATH="/usr/local;/opt/homebrew;/opt/local"'
+cmakeargs = '-DCMAKE_IGNORE_PREFIX_PATH="/usr/local;/opt/homebrew;/opt/local" -DCMAKE_FIND_FRAMEWORK=LAST'

 # Extra meson args. --wrapmode=nofallback prevents meson projects from
 # building dependencies on their own. We generally want failed
